Dockerfile

```diff
@@ -1,21 +1,19 @@
-FROM node:latest
-
-ENV HOME=/home/blue-stream
-
-COPY package*.json $HOME/app/
-
-# RUN chown -R node $HOME/* /usr/local/
-
-WORKDIR $HOME/app
-
-RUN npm install --silent --progress=false
-
-COPY . $HOME/app/
-
-# RUN chown -R node $HOME/*
-
-EXPOSE 3000
-
-# USER node
-
+FROM node:10.15-alpine
+ENV NODE_ENV=development
+ENV HOME=/usr/src/app
```

Comment:
Redundant step, no need to define this as a variable and COPY should be done with “. ./“ because WORKDIR already puts you in that directory like “cd”